packages/material-ui-styles/src/defaultTheme/defaultTheme.spec.ts

```diff
@@ -9,6 +9,7 @@ declare module '@material-ui/styles' {
 }
 
 {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
nit: convert block to function so that we're safe.

Reply:
OK